client/src/main/java/io/confluent/kafka/schemaregistry/rules/DlqAction.java

```diff
@@ -53,7 +53,6 @@ public class DlqAction implements RuleAction {
   public static final String RULE_TOPIC = HEADER_PREFIX + "topic";
```

Comment:
Should we fix this too?

Reply:
Good question.  This is the original (source) topic, not the DLQ topic